packages/expo-media-library/ios/EXMediaLibrary/EXMediaLibrary.m

```diff
@@ -388,19 +388,31 @@ - (NSDictionary *)constantsToExport
   
   if (asset) {
     NSMutableDictionary *result = [EXMediaLibrary _exportAssetInfo:asset];
-    PHContentEditingInputRequestOptions *options = [PHContentEditingInputRequestOptions new];
-    options.networkAccessAllowed = YES;
-    
-    [asset requestContentEditingInputWithOptions:options completionHandler:^(PHContentEditingInput * _Nullable contentEditingInput, NSDictionary * _Nonnull info) {
-      result[@"localUri"] = [contentEditingInput.fullSizeImageURL absoluteString];
-      result[@"orientation"] = @(contentEditingInput.fullSizeImageOrientation);
-      
-      if (asset.mediaType == PHAssetMediaTypeImage) {
+    if ([result[@"mediaType"] isEqual:@"photo"]) {
+      PHContentEditingInputRequestOptions *options = [PHContentEditingInputRequestOptions new];
+      options.networkAccessAllowed = YES;
+
+      [asset requestContentEditingInputWithOptions:options
+                                 completionHandler:^(PHContentEditingInput * _Nullable contentEditingInput, NSDictionary * _Nonnull info) {
+        result[@"localUri"] = [contentEditingInput.fullSizeImageURL absoluteString];
+        result[@"orientation"] = @(contentEditingInput.fullSizeImageOrientation);
+        
         CIImage *ciImage = [CIImage imageWithContentsOfURL:contentEditingInput.fullSizeImageURL];
         result[@"exif"] = ciImage.properties;
-      }
-      resolve(result);
-    }];
+        resolve(result);
+      }];
+    } else {
+      PHVideoRequestOptions *options = [PHVideoRequestOptions new];
+      options.networkAccessAllowed = YES;
+
+      [[PHImageManager defaultManager] requestAVAssetForVideo:asset
+                                                      options:options
+                                                resultHandler:^(AVAsset * _Nullable asset, AVAudioMix * _Nullable audioMix, NSDictionary * _Nullable info) {
+        AVURLAsset *urlAsset = (AVURLAsset *)asset;
+        result[@"localUri"] = [[urlAsset URL] absoluteString];
```

Comment:
Is there anything else available in AVURLAsset that we can export to JS?
Also, it might be worth to update the documentation as well that we don't provide `orientation` when the `mediaType` is different than `photo`

Reply:
`AVUrlAssets` doesn't provide anything useful.

In our documentation, in the description column of `orientation` property, we have this:
`Display orientation of the image`. Is it not enough?

Reply:
I'd rephrase it to make it clearer for users: 
> Orientation is available only for assets whose `mediaType` is `MediaType.photo`
Also, maybe we can infer the orientation based on the video dimensions? 🤔